release-notes/opensearch-geospatial.release-notes-2.10.0.0.md

```diff
@@ -0,0 +1,22 @@
+## Version 2.10.0.0 Release Notes
+
+Compatible with OpenSearch 2.10.0
+
+### Features
+* IP2Geo processor implementation ([#362](https://github.com/opensearch-project/geospatial/pull/362))
```

Comment:
We should have better naming for this. Impact does not show up. @heemin32

Reply:
imo, release log is not a place where impact can be seen.